libvast/vast/aliases.hpp

```diff
@@ -15,12 +15,11 @@
 
 #include <cstdint>
 #include <limits>
-#include <map>
 #include <memory>
-#include <set>
```

Comment:
This commit seems to do more than what it describes. In the future, please keep the commits more-or-less contained or point out non-related changes.